pkg/projects/Microsoft.NETCore.App/netcoreapp1.0.props

```diff
@@ -5,10 +5,6 @@
       <Version>1.3.0</Version>
       <Exclude>Compile</Exclude>
     </NETCoreApp10Dependency>
-    <NETCoreApp10Dependency Include="Microsoft.CodeAnalysis.VisualBasic">
```

Comment:
@ericstj I removed these for netcoreapp10/11 to have the conversation but I can imagine we keep these for back compat.